src/flow_graph.py

```diff
@@ -641,19 +686,19 @@ def find_block_by_label(label: str) -> Optional[Block]:
                         arg = arg.lhs
                     if (
                         isinstance(arg, Macro)
-                        and arg.macro_name == "lo"
+                        and arg.macro_name in ("lo", "l")
                         and isinstance(arg.argument, AsmGlobalSymbol)
                         and any(
                             arg.argument.symbol_name.startswith(prefix)
-                            for prefix in ("jtbl", "jpt_")
+                            for prefix in ("jtbl", "jpt_", "lbl_")
                         )
                     ):
                         jtbl_names.append(arg.argument.symbol_name)
             if len(jtbl_names) != 1:
                 raise DecompFailure(
                     f"Unable to determine jump table for {jump.mnemonic} instruction {jump.meta.loc_str()}.\n\n"
                     "There must be a read of a variable in the same block as\n"
-                    'the instruction, which has a name starting with "jtbl"/"jpt_".'
+                    'the instruction, which has a name starting with "jtbl"/"jpt_"/"lbl_".'
                 )
 
             jtbl_name = jtbl_names[0]
```

Comment:
the `is_constant_branch = jump.mnemonic in ["b", "j"]` check further down in this file would be good to arch-ify